Maquetado/scss/titles.scss

```diff
@@ -0,0 +1,16 @@
+$urlImagen: "./assets/triangle.svg";
+
+.front-end{
+  background-image: url($urlImagen);
+  background-repeat: no-repeat;
+  margin: 50px 0 0 100px;
+  padding: 45px 62px 65px;
```

Comment:
One of the most important subjects while coding is to write code for others, not only for readability but to be easy to modify it. Maybe this is not the most clearable way to understand this effect, i can suggest you maybe using the [::before css selector](https://developer.mozilla.org/en-US/docs/Web/CSS/::before) or could it as simple as making an image tag and using layout 'magic' to be more clear with what you want to achieve.